crates/ide-completion/src/completions/postfix.rs

```diff
@@ -186,6 +200,13 @@ pub(crate) fn complete_postfix(
                 &format!("match {receiver_text} {{\n    ${{1:_}} => {{$0}},\n}}"),
             )
             .add_to(acc, ctx.db);
+
+            postfix_snippet(
+                "lete",
+                "let pat = expr else {}",
+                &format!("let $1 = {receiver_text} else {{\n    $0\n}};"),
+            )
+            .add_to(acc, ctx.db);
```

Comment:
I think it's a irrefutable `let...else` pattern, like `let foo = 0 else { return; }`

Reply:
One is able to match against any pattern they wish, like match, 
`let Enum::Variant(val)` = ... else ...`